skimage/feature/blob.py

```diff
@@ -25,26 +26,27 @@ def _blob_overlap(blob1, blob2):
     Parameters
     ----------
     blob1 : sequence
-        A sequence of ``(y,x,sigma)``, where ``x,y`` are coordinates of blob
-        and sigma is the standard deviation of the Gaussian kernel which
-        detected the blob.
+        A sequence of ``(r,c,sigma)`` or ``(f,r,c,sigma)``, where ``r,c``
+        (or ``(f,r,c,sigma)``) are coordinates of blob and ``sigma`` is
+        the standard deviation of the Gaussian kernel which detected the blob.
```

Comment:
I agree. Could we ask here for `ndarray` instead (same as in the functions below)?